docs/shared/contribution_guidelines/vocabulary.rst

```diff
@@ -4,10 +4,15 @@ Vocabulary
 
 If you want to contribute to openMINDS by adding information to the openMINDS vocabulary we advise you to first review this chapter.
 
+For each contribution, please raise an `ISSUE <https://github.com/openMetadataInitiative/openMINDS/issues>`_. Start the issue title with ``vocab: `` followed by an informative short title for your contribution. Please provide a more detailed explanation of your contribution in issue description. We will discuss your contribution with you before a pull request is made (by you or by us if you do not feel comfortable making a pull request yourself).
```

Comment:
perhaps we could add some issue templates, starting with one for vocabulary contributions?

Reply:
I like this suggestion. These would go though into a different PR on the other repos. I've created an issue: https://github.com/openMetadataInitiative/openMINDS/issues/32 so that we don't forget.